### PR TITLE
FW-2276 Use KM API URL for bot detail | web plugin

### PR DESCRIPTION
**Pull request by Ojashwa**
### What do you want to achieve?
-

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [x] I have tested it in IE Browser.

### How was the code tested?

- **use installation script and check for the API details in network panel**
---
image
---
image

 

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Review by devashishmamgain — APPROVED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 1]
```diff
@@ -12695,14 +12694,20 @@ var userOverride = {
                 return conversationDetail;
             };
             _this.checkBotDetail = function (userId) {
-                window.Applozic.ALApiService.ajax({
+                var authKey = JSON.parse(
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:12697]: Is this the same which is used in any other api?

Comment (Ojashwa) [reply webplugin/js/app/mck-sidebox-1.0.js:12697]: yes.

Comment (devashishmamgain) [reply webplugin/js/app/mck-sidebox-1.0.js:12697]: can you share any code reference for that

Comment (Ojashwa) [reply webplugin/js/app/mck-sidebox-1.0.js:12697]: how the headers are being set?

Comment (devashishmamgain) [reply webplugin/js/app/mck-sidebox-1.0.js:12697]: I mean any other place from where auth header is fetched from storage

**Review by Rudranand — DISMISSED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 1 above)
Comment (Ojashwa) [reply webplugin/js/app/mck-sidebox-1.0.js]: Ratified

**Review by devashishmamgain — APPROVED** [review]

**Review by Rudranand — APPROVED** [review]

**Pull request merged**